isos/bootstrap/bootstrap

```diff
@@ -39,7 +39,7 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     # WARNING WARNING WARNING WARNING WARNING
     # if the tmpfs is not large enough odd hangs can occur and the ESX event log will
     # report the guest disabling the CPU
-    mount -t tmpfs -o size=64m tmpfs ${MOUNTPOINT}/.tether/
+    mount -t tmpfs -o size=128m tmpfs ${MOUNTPOINT}/.tether/
```

Comment:
can you check to see what size you actually need this to be, then set it to that plus maybe 10% headroom?
Use `df -h` in a containerVM and you can see the consumption

Reply:
Absolutely.